recipes/caffe2/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "$PREFIX" ]; then
+  PREFIX="$CONDA_PREFIX"
+fi
+
+# conda build will copy everything over, including build directories.
+# Don't let this pollute the build!
+rm -rf build || true
+
+PYTHON_ARGS="$(python ./scripts/get_python_cmake_flags.py)"
+
+mkdir -p build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX="$PREFIX" -DCMAKE_PREFIX_PATH="$PREFIX" $PYTHON_ARGS ..
+make -j20
```

Comment:
This is silly on the CIs, which only have 2 cores. `make -j$CPU_COUNT` would be better.